webboot/webboot.go

```diff
@@ -234,12 +243,33 @@ func main() {
 
 	}
 	if *dryrun == false {
-		bookmark[filename].Cmdline = string(cl) + " " + bookmark[filename].Cmdline
+		k, i := wbpath(tmp, b.Kernel), wbpath(tmp, b.Initrd)
+		cmdline := string(cl) + " " + b.Cmdline
 
-		if err := mountkexec.KexecISO(bookmark[filename], tmp); err != nil {
-			log.Fatalf("Error in kexecISO:%v", err)
+		kernel, err := os.Open(k)
+
+		if err != nil {
+			log.Fatal(err)
+		}
+		var image boot.OSImage
+		if err := multiboot.Probe(kernel); err == nil {
+			image = &boot.MultibootImage{
```

Comment:
This image can't be right

Reply:
It's in the original; shall I just yank it? What is not right.

Reply:
I mean the original webboot stuff. That's almost certainly wrong. I'm inclined to removed this until we care about multiboot.

Reply:
I remove dit ...